src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/request/enums/TaskOperationType.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers.request.enums;
+
+public enum TaskOperationType {
+    MARK_TO_RECONFIGURE("mark_to_reconfigure", false),
+    EXECUTE_RECONFIGURE("execute_reconfigure", true);
```

Comment:
Maybe add a comment here for devs who may add a new operation in future.  I guess this boolean is to determine whether the operation requires a re-indexing of signatures, which for new devs may be unclear since it is complex.

Reply:
This is not a blocker for this PR but ... It would be nice to be able to abstract this complexity somehow as most new devs don't have to worry about it.  Is there a default that makes sense?  E.g. it's always true unless you specifically opt out?  Maybe that will just mean redundant re-indexing.  Or perhaps we could have some type of meaningful operation category (Task/Role update vs no Task / Role update) which means that the indexing happens or not.

Reply:
Added a comment to explain boolean flag. As you have said, we can add operation types in a different ticket.